src/components/footer/Footer.tsx

```diff
@@ -25,28 +25,30 @@ const SocialMediaList = [
 ]
 
 const Footer: React.FC = () => (
-  <Container as="footer" textAlign="center" paddingTop={14} paddingBottom={12}>
-    <Stack spacing="5">
-      <Stack justifyContent="center" direction="row" align="center">
-        <ButtonGroup variant="ghost">
-          {SocialMediaList.map((socialmedia: SocialMedia) => (
-            <IconButton
-              key={socialmedia.name}
-              isRound
-              as="a"
-              href={socialmedia.link}
-              aria-label={socialmedia.name}
-              icon={<socialmedia.icon fontSize="1.25rem" />}
-            />
-          ))}
-        </ButtonGroup>
+  <ChakraProvider>
```

Comment:
The ChakraProvider shouldn't be included in each component per the docs [here](https://www.npmjs.com/package/@chakra-ui/provider).  As such, just include it in each `stories.tsx` file.